python/ray/autoscaler/v2/utils.py

```diff
@@ -194,7 +374,10 @@ def _parse_cluster_resource_usage(
         cluster_resource_usage = defaultdict(ResourceUsage)
 
         for node_state in state.node_states:
-            cls._parse_node_resource_usage(node_state, cluster_resource_usage)
+            if node_state.status != NodeStatus.DEAD:
```

Comment:
what's the reason behind this change? I think it is probably better still having cluster resources of the dead node?

Reply:
hmm, but a dead node technically doesn't contribute to any cluster resources? (e.g. it will be confusing to see 10 CPUs total while all 10s are from a dead node? As the underlying instance might also be gone)

Reply:
Are we showing this from the ray status? (individual dead node)?

Reply:
yes - not resources usage. but node (ip, node_id, type) as failed nodes.